### PR TITLE
Ap is

### PR DESCRIPTION
**Pull request by rivera-fernando**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/deleteAll.java [hunk 1]
```diff
@@ -36,13 +38,16 @@
...
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         Query query = new Query("Comment");
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         PreparedQuery results = datastore.prepare(query);
-        for (Entity entity : results.asIterable()) {
+        if (userService.isUserLoggedIn() && userService.isUserAdmin()) {
+            for (Entity entity : results.asIterable()) {
             Key key = entity.getKey();
             datastore.delete(key);
+            }
```
Comment (serzhk) [inline portfolio/src/main/java/com/google/sps/servlets/deleteAll.java]: Indentation?

File: portfolio/src/main/webapp/materialize.js [hunk 2]
```diff
@@ -12546,3 +12555,205 @@ async function sendComment() {
...
+  initMap()
+  document.cookie = "dark=yes; path=/";
+} else if (dark_mode == "yes"){
+  createMap();
+  document.cookie = "dark=no; path=/";
+} else {
+  initMap();
+  document.cookie = "dark=yes; path=/";
+}
+}
```
Comment (serzhk) [inline portfolio/src/main/webapp/materialize.js]: new line at the end

**Review by serzhk — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/deleteAll.java (hunk 1 above)
Comment (rivera-fernando) [reply portfolio/src/main/java/com/google/sps/servlets/deleteAll.java]: done

File: portfolio/src/main/webapp/materialize.js (hunk 2 above)
Comment (rivera-fernando) [reply portfolio/src/main/webapp/materialize.js]: done

**Review by serzhk — APPROVED** [review]

**Pull request merged**